osf/management/commands/manage_switch_flags.py

```diff
@@ -1,38 +1,34 @@
 # -*- coding: utf-8 -*-
 import logging
+import yaml
 
 from django.core.management.base import BaseCommand
 
-from osf.features import switches, flags
-from waffle.models import Flag, Switch
-
 logger = logging.getLogger(__name__)
 
-def manage_waffle(delete_waffle=False):
-    file_switches = list(switches.values())
-    current_switches = Switch.objects.values_list('name', flat=True)
 
-    add_switches = set(file_switches) - set(current_switches)
-    for switch in add_switches:
-        Switch.objects.get_or_create(name=switch, defaults={'active': False})
-        logger.info('Adding switch: {}'.format(switch))
+def manage_waffle(delete_waffle=False):
+    from django.apps import apps
 
-    file_flags = list(flags.values())
-    current_flags = Flag.objects.values_list('name', flat=True)
+    Flag = apps.get_model('waffle.Flag')
+    Switch = apps.get_model('waffle.Switch')
 
-    add_flags = set(file_flags) - set(current_flags)
-    for flag_name in add_flags:
-        Flag.objects.get_or_create(name=flag_name, defaults={'everyone': False})
-        logger.info('Adding flag: {}'.format(flag_name))
+    with open('osf/features.yaml', 'r') as stream:
+        features = yaml.safe_load(stream)
+    for flag in features['flags']:
+        flag.pop('flag_name')
+        Flag.objects.get_or_create(name=flag['name'], defaults=flag)
+    for switch in features['switches']:
+        switch.pop('flag_name')
+        Switch.objects.get_or_create(name=switch['name'], defaults=switch)
```

Comment:
@jwalz I want to draw your attention to fact that I'm changing this to `update_or_create` I think that's probably the assumed behavior, not much point in just getting the flag or switch if the config changes.